src/index.mjs

```diff
@@ -19,14 +19,12 @@ import requestIdleCallback from './request-idle-callback.mjs';
 
 const toPrefetch = new Set();
 
-const observer = new IntersectionObserver(entries => {
-  entries.forEach(entry => {
-    if (entry.isIntersecting) {
-      const url = entry.target.href;
-      if (toPrefetch.has(url)) prefetcher(url);
-    }
-  });
-});
+const observerConfig = {
```

Comment:
Let's remove this – as is, this just redefines the existing defaults without a way to customize it.